.changeset/odd-bottles-fix.md

```diff
@@ -0,0 +1,10 @@
+---
+'@sl-design-system/tooltip': patch
+'@sl-design-system/shared': patch
+---
+
+Various fixes:
+- Make it possible to close a tooltip with Escape key
+- Fixed showing a tooltip not only once
+- Fixed showing shared tooltip
+- Fixed tooltip accessibility - removed `aria-expaned` from a tooltip component
```

Comment:
Typo in `aria-expanded`; also can you explain why removing `aria-expanded` fixes tooltip a11y?